### DISCUSSION TITLE
LD-006 and UV-Tools - Settings are "locked" on printer settings display ?

### DISCUSSION THREAD

**Discussion by azalanono** [sn4k3/UVtools#939] [General]
Hello.
As mentioned in title, I own a Creality LD-006 printer. I create the .ctb file with Chitubox 2.0 Basic, as I always do.

Because the lifting height was too low (and the FEP film didn't separate therefore, I used UV-Tools to increase the lifting height to 12 mm, also changed the rising speed, then I saved the file.

I was very astonished that I cannot change this few parameters available in the settings menu on the printer, while this is possible when not using UV-TOOLS.

Is there a way to get them editable ?
Especially curing times would be very useful if I can change them during print when I notice I should increase/decrease curing time.

**Comment by sn4k3** [comment]
When editing with UVtools and apply any suggestions it will enable per layer settings which disable global settings and also edits from printer LCD.
Try not to apply the suggestions and edit only the global settings, it should work that way...

**Reply by azalanono** [reply]
Hmm - This is very nasty.
Is there no way to prevent that in UV-Tool Settings?  I mean something like "Don't change ever to "per layer settings", prevent some changes instead.

**Reply by sn4k3** [reply]
That flag is not changed with default settings.
It can be due any suggestion (Shield icon/tab) you apply manually or auto, check if is the case.
If you edit layer parameter individually the flag will also turn on.

In any case you can also reset layer properties on "File" menu which reverts all to globals and turn flag off.
